test/tests/serial/mail_with_ldap.bats

```diff
@@ -158,6 +158,53 @@ function teardown_file() {
   done
 }
 
+
+@test "dovecot: ldap email listing works" {
+  _run_in_container doveadm user '*'
+  assert_success
+  assert_line --index 0 'some.user@localhost.localdomain'
+  assert_line --index 1 'some.other.user@localhost.otherdomain'
+  assert_line --index 2 'some.user.email@localhost.localdomain'
+  _should_output_number_of_lines 3
+}
+
+@test "dovecot: (ENV DOVECOT_ITERATE_ATTRS=uniqueIdentifier=user) ldap email listing works" {
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env DOVECOT_ITERATE_ATTRS='uniqueIdentifier=user'
+  )
```

Comment:
for reasons that I ignore, this does not seem to give the options to the container creation process

Any help would be very welcome 🙏🏻

Reply:
You will need to provide the array `CUSTOM_SETUP_ARGUMENTS` on container startup to the function that creates the container. Functions like `_run_in_container` do not take it into account.

If you want to have it added to the ENV of the container, you'll need to do so at the top of the file where the container is created I guess (or wherever it is created). Probably

https://github.com/docker-mailserver/docker-mailserver/pull/3426/files#diff-664e916443d73f328b2c0c61c3a1aca1de5b968cfb03316012765971cbb5980bR79

and

https://github.com/docker-mailserver/docker-mailserver/pull/3426/files#diff-664e916443d73f328b2c0c61c3a1aca1de5b968cfb03316012765971cbb5980bR93